webbpsf/opds.py

```diff
@@ -126,8 +130,19 @@ def __init__(self, name='unnamed OPD', opd=None, opd_index=0, transmission=None,
 
         self.segnames = np.asarray([a[0:2] for a in constants.SEGNAMES_WSS_ORDER])
 
-        full_seg_mask_file = os.path.join(utils.get_webbpsf_data_path(), segment_mask_file)
+        if segment_mask_file == 'JWpupil_segments.fits' and npix != 1024: # Be backwards compatible with file naming
```

Comment:
> # Be backwards compatible with file naming

FWIW I don't think this is particularly important to keep exactly backwards compatible. It would simplify things to rename the existing file to have a 1024 in the name, so shall we just do that so they're all consistent?

Reply:
Sure, I can make that change in the code. Do you want me to change the name of the one in tel/share/webbpsf/webbpsf-data-source?

Reply:
Sure. 

I'm trying to recall why I did not change the filename before. I think in part I was trying to minimize the need for new data files for use by the CI runner - switching the expected name will immediate make the CI system fail since it can't find the new name in the existing data files. But we need to update that now anyway as part of this so I think that's fine.

Reply:
Sure. 

I'm trying to recall why I did not change the filename before. I think in part I was trying to minimize the need for new data files for use by the CI runner - switching the expected name will immediate make the CI system fail since it can't find the new name in the existing data files. But we need to update that now anyway as part of this so I think that's fine.